src/main/scala/uk/gov/nationalarchives/dp/client/DataProcessor.scala

```diff
@@ -288,6 +288,20 @@ class DataProcessor[F[_]]()(implicit me: MonadError[F, Throwable]) {
         )
       }
     }
+
+  /** Returns a `Float`
+    * @param elem
+    *   The element containing the namespace with the version in it
+    * @return
+    *   A `Float` parsed from the XML
```

Comment:
Can you add a description of what the float is, e.g. `A float representing the Preservica version` or something like that.